src/nagini_translation/extended_ast_translator.py

```diff
@@ -0,0 +1,68 @@
+"""
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""
+
+from nagini_translation.extended_ast.translators.statement import \
+    ExtendedASTStatementTranslator
+from nagini_translation.extended_ast.translators.method import ExtendedASTMethodTranslator
+from nagini_translation.lib.jvmaccess import JVM
+from nagini_translation.lib.typeinfo import TypeInfo
+from nagini_translation.lib.viper_ast import ViperAST
+from nagini_translation.translator import Translator
+from nagini_translation.translators.abstract import TranslatorConfig
+from nagini_translation.translators.call import CallTranslator
+from nagini_translation.translators.contract import ContractTranslator
+from nagini_translation.translators.expression import ExpressionTranslator
+from nagini_translation.translators.io_operation import IOOperationTranslator
+from nagini_translation.translators.obligation import ObligationTranslator
+from nagini_translation.translators.permission import PermTranslator
+from nagini_translation.translators.predicate import PredicateTranslator
+from nagini_translation.translators.program import ProgramTranslator
+from nagini_translation.translators.pure import PureTranslator
+from nagini_translation.translators.type import TypeTranslator
+from nagini_translation.translators.type_domain_factory import \
```

Comment:
```
from nagini_translation.translators.type_domain_factory import (
    TypeDomainFactory
)
```

Reply:
(Generally, try to parenthesize things to avoid the backslashes at the end of lines)